### ISSUE TITLE
Ligatures like æ

### ISSUE THREAD

**Issue by ariutta** [orsinium/homoglyphs#12]
Hi,

I have another one :-) This time it's ligatures like `æ`.

```py
import homoglyphs as hg
hg_instance = hg.Homoglyphs(categories=hg.Categories.get_all(), strategy=hg.STRATEGY_LOAD, ascii_strategy=hg.STRATEGY_REMOVE)
hg_instance.get_combinations('æ')
```

Actual: `['æ']`

Expected: `['ae', 'ӕ', 'æ']` ([from Unicode Utilities: Confusables](https://unicode.org/cldr/utility/confusables.jsp?a=%C3%A6&r=None))

I think `to_ascii` should return `['ae']`.

**Comment by ariutta** [comment]
Yes, it appears `to_ascii` is limiting results to 256-characters sets: [core.py#L233](https://github.com/orsinium/homoglyphs/blob/11d20c4b6cf3d65d0529c44fd4252822a5e317d6/homoglyphs/core.py#L233). I'll move this item to a separate issue.

**Comment by orsinium** [comment]
:thinking:

**Comment by orsinium** [comment]
Ok, this char is presented in `confusables.json`, so we have to discover `core.py`

**Comment by orsinium** [comment]
Wow, I know. This is because `fi` not in alphabet:

```python
'fi' in hg.Categories.get_alphabet(hg.Categories.get_all())
```
